src/hoc/__test__/CreditsProvider.test.tsx

```
@@ -71,86 +90,19 @@ describe('CreditsProvider', () => {
-        const { supabase } = require('@/common/supabase');
-        vi.mocked(supabase.auth.getSession).mockResolvedValue({
-            data: {
-                session: {
-                    access_token: 'fake-token',
-                },
-            },
-        });
-
-        // Mock axios error
+        // Mock axios error correctly
         const axios = require('axios');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)